tst/standard/attr.tst

```diff
@@ -192,6 +192,12 @@ gap> AdjacencyMatrix(Digraph(rec(DigraphNrVertices := 0,
 >                                DigraphRange      := [])));
 [  ]
 
+#  DigraphNrAdjacencies
+gap> G := RandomDigraph(50);;
```

Comment:
Better not use random graphs in the tests, if this fails for some reason we won't have any idea what the input graph was

Reply:
Is there a standard replacement for this, or should I just hardcode my own graph?

Reply:
Best to hard-code a return value of RandomDigraph(50) that you've found

Reply:
I opted for `RandomDigraph(30)` so the input wasn't so unruly, but otherwise that's done now.